python-package/xgboost/dask.py

```diff
@@ -90,6 +90,11 @@ def __exit__(self, *args):
 
 def concat(value):              # pylint: disable=too-many-return-statements
     '''To be replaced with dask builtin.'''
+    def unbox_0d_array(x):  # pylint: disable=invalid-name
+        if isinstance(x, numpy.ndarray) and x.dtype == numpy.object and x.shape == ():
```

Comment:
Is it an empty array?  If so can we simply filter it out?

Reply:
No, it's a zero-dimensional array with a single element in it. It's not the same as an empty array.